src/loaders/svg/tvgSvgUtil.cpp

```diff
@@ -46,7 +46,7 @@ size_t svgUtilURLDecode(const char *src, char** dst)
     auto length = strlen(src);
     if (length == 0) return 0;
 
-    char* decoded = (char*)malloc(sizeof(char) * length + 1);
+    char* decoded = (char*)calloc(length + 1, sizeof(char));
     decoded[length] = '\0';
```

Comment:
It could count the actual length of the decoded string (=> idx?). 
For example, this approach might be more efficient than unnecessarily clearing memory.
```
 decoded[idx] = '\0';
```

Reply:
you are right. I hadn't thought about return size. I updated PR .